CMakeLists.txt

```diff
@@ -220,6 +220,8 @@ target_link_libraries(celerity_runtime PUBLIC
   Threads::Threads
   MPI::MPI_CXX
   spdlog::spdlog
+  $<$<AND:$<BOOL:${WIN32}>,$<CONFIG:Debug>>:sycld>
+  $<$<AND:$<BOOL:${WIN32}>,$<CONFIG:Release>>:sycl>
```

Comment:
This looks DPC++ specific, maybe wrap in a `if(CELERITY_SYCL_IMPL STREQUAL "DPC++")`?

Reply:
I'm not sure if this is DPC++ or windows specific, and it's hard to test. But I wrapped it anyway, and this check can be extended later if need be.